<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>publicTransportRunnerConfig</class>
 <widget class="QWidget" name="publicTransportRunnerConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>196</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>231</width>
    <height>142</height>
   </size>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblServiceProvider">
     <property name="text">
      <string>Service Provider:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="serviceProvider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">---</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="btnChangeStop">
       <property name="text">
        <string>&amp;Change...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblDepartureKeyword">
     <property name="text">
      <string>&amp;Departure Keyword:</string>
     </property>
     <property name="buddy">
      <cstring>departureKeyword</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KLineEdit" name="departureKeyword"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblArrivalKeyword">
     <property name="text">
      <string>&amp;Arrival Keyword:</string>
     </property>
     <property name="buddy">
      <cstring>arrivalKeyword</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KLineEdit" name="arrivalKeyword"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblJourneyKeyword">
     <property name="text">
      <string>&amp;Journey Keyword:</string>
     </property>
     <property name="buddy">
      <cstring>journeyKeyword</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KLineEdit" name="journeyKeyword"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblStopsKeyword">
     <property name="text">
      <string>&amp;Stop Suggestion Keyword:</string>
     </property>
     <property name="buddy">
      <cstring>stopsKeyword</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KLineEdit" name="stopsKeyword"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblResultCount">
     <property name="text">
      <string>Maximal Result Count:</string>
     </property>
     <property name="buddy">
      <cstring>stopsKeyword</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="KIntNumInput" name="resultCount">
     <property name="value">
      <number>4</number>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="referencePoint">
      <number>1</number>
     </property>
     <property name="suffix">
      <string> results</string>
     </property>
     <property name="specialValueText">
      <string>1 result</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnChangeStop</tabstop>
  <tabstop>departureKeyword</tabstop>
  <tabstop>arrivalKeyword</tabstop>
  <tabstop>journeyKeyword</tabstop>
  <tabstop>stopsKeyword</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
